src/libs/fileDownload/index.android.js

```diff
@@ -45,17 +45,26 @@ function handleDownload(url, fileName) {
             path: `${path}/${attachmentName}`,
             addAndroidDownloads: {
                 useDownloadManager: true,
-                notification: true,
+                notification: false,
```

Comment:
Can you remind me why? IIRC this wasn't an option with MediaStorage. Do we even need this addAndroidDownloads config still?

Reply:
I'm still seeing the notification, even with this flag turned off 😕